api_tests/preprints/views/test_preprint_list_mixin.py

```diff
@@ -0,0 +1,156 @@
+from nose.tools import *  # flake8: noqa
+
+from api.base.settings.defaults import API_BASE
+
+from website.util import permissions
+
+from osf_tests.factories import (
+    ProjectFactory,
+    PreprintFactory,
+    AuthUserFactory,
+    SubjectFactory,
+)
+
+from api_tests import utils as test_utils
+
+class PreprintIsPublishedListMixin(object):
+
+    def setUp(self):
+        super(PreprintIsPublishedListMixin, self).setUp()
```

Comment:
This is fine for now, but in the future, we should be using pytest fixtures for this sort of thing. Using fixtures is more composable and can potentially lead to faster tests (because you only set up the objects that you need for a particular test).